lambda_functions/functions/synchronisation/go.mod

```diff
@@ -0,0 +1,20 @@
+module synchronisation
+
+go 1.17
```

Comment:
This says 1.17 but in the Dockerfile we use 1.18 -- should we change to make sure they correlate?